src/main/resources/config/application.yml

```diff
@@ -10,7 +10,6 @@ security:
       tokenInfoUri: ${TOKENINFO_URL}
 
 management:
-  port: 7979
```

Comment:
yeah, management port is needed (we obviously could alternatively set it in deployment descriptor, i.e. env vars)